core/common/src/Gradient.ts

```diff
@@ -300,13 +300,11 @@ export namespace Gradient {
       const stepCount = Math.min(settings.stepCount, maxDimension);
       const dimension = (ThematicGradientMode.Smooth === settings.mode) ? maxDimension : stepCount;
       const hasAlpha = this.hasTranslucency;
```

Comment:
You could simplify things by getting rid of the `hasAlpha` check...if it is false then `color.getAlpha` below is going to return 255 for every color.

Reply:
Thanks, did not realize that.